### ISSUE TITLE
Redirects from /page/x to /x which causes 404

### ISSUE THREAD

**Issue by CrossScarDev** [iamcco/markdown-preview.nvim#685]
**Describe the bug**
When opening the Markdown Preview it redirects from `/page/x` to `/x`. This causes an error on refresh and other forms of navigation including using the backwards and forwards arrows in the browser: #625.

**To Reproduce**
1. Run `:MarkdownPreview`
2. Open Browser
3. Refresh Page

**Expected behavior**
I would expect the page not to get redirected at all as the content is at `/page/x`.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Arch Linux

Log: N/A

**Comment by JosiahWayne** [comment]
Same issue, also looking for a solution.

**Comment by CrossScarDev** [comment]
It looks like this is no longer maintained, given the fact that the last update was 2 years ago, though I can't seem to find an alternative solution.

**Comment by JosiahWayne** [comment]
Yes, I also find that this repo is no longer maintained, I found this for alternative, you may check it out: https://github.com/jannis-baum/Vivify

**Comment by JosiahWayne** [comment]
It is a file previewer and has a nvim plugin.

**Comment by CrossScarDev** [comment]
Thank you for finding this.

**Comment by CrossScarDev** [comment]
I'm going to close this as not planned as this is no longer maintained
